docs/api.rst

```diff
@@ -5,6 +5,8 @@ API Reference
 Basic Exploration
 *****************
 
+.. automethod:: api.client.gro_client.GroClient.find_data_series
```

Comment:
The url in the "See also" section of the docstring isn't formatting nicely. Might need to put it in angle brackets? `<https://github.com/gro-intelligence/api-client/wiki/Data-Series-Definition>`? Can play around with that

Reply:
I've updated the url to https://developers.gro-intelligence.com/data-series-definition.html in my fix-docs-links PR

Reply:
But it's not in this webpage in your PR since it's being added here. Didn't realize it renders weird when it's in the "See also" section (see https://developers.gro-intelligence.com/nemozen-patch-1/api.html ). Looks fine elsewhere in https://developers.gro-intelligence.com/fix-docs-links/api.html because we didn't have any instances of it being in "See also"

Reply:
Testing here to see if the angle brackets fix it: https://github.com/gro-intelligence/api-client/pull/140/commits/8080f1a9bac15845e3e6d6748cb9c82105cda8ef

Reply:
https://github.com/gro-intelligence/api-client/pull/140/commits/8a71aec167dc4577f7ab3dff3833a53afc0a4d5a this works to make it a link, but it messes up the `get_data_series()` format. Will just move the link up to the main description instead.

Reply:
https://github.com/gro-intelligence/api-client/pull/140/commits/3099c393d67e8c2870d1086606673112756b5abd done. This looks better.